spec/integration/_dev/build/build.spec.yml

```
@@ -25,6 +25,7 @@ spec:
               description: >
                 Whether or not import common used dynamic templates and properties into the package
               default: false
+              deprecated: true
```

Nit. Consider adding a comment here with a link to this PR or some other URL with more information about this deprecation.
```suggestion
              deprecated: true # See: https://github.com/elastic/package-spec/pull/726
```